Ports/libjodycode/package.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='libjodycode'
+version='3.1'
+files="https://github.com/jbruchon/libjodycode/archive/refs/tags/v${version}.tar.gz libjodycode-${version}.tar.gz c72974eb1d38873e06ea84b3d78990f87192f0113da5bd13fcac6bbc6a6e2184"
+auth_type='sha256'
+workdir="libjodycode-${version}"
```

Comment:
Given this: 
https://github.com/jbruchon/libjodycode/blob/master/Makefile#L26

this package will behave particularly badly if we try to cross-compile from aarch64-macOS to x86_64-serenity.

we should be able to override `UNAME_M`, `UNAME_S` and `CROSS_DETECT` in `makeopts()` though.

Reply:
Alright, I've overridden these values and everything compiles like before. Should I make any more modifications?